### PR TITLE
Converted OpenSourceApp.py file name to Open_Source_App.py

### PR DESCRIPTION
**Pull request by subhajit20**
## Description
Converted OpenSourceApp.py file name to Open_Source_App.py
Issue - #666 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (does not change functionality, e.g. code style improvements, linting)
- [ ] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please delete options that are not relevant.

- [ ] Unit Test
- [ ] Test Script (please provide)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] Made sure Checks passed

### CODE REVIEW

**Comment by subhajit20** [comment]
@taranjeet please check and let me know for further corrections.

File: embedchain/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from embedchain.apps.App import App  # noqa: F401
 from embedchain.apps.CustomApp import CustomApp  # noqa: F401
 from embedchain.apps.Llama2App import Llama2App  # noqa: F401
-from embedchain.apps.OpenSourceApp import OpenSourceApp  # noqa: F401
+from embedchain.apps.Open_Source_App import OpenSourceApp  # noqa: F401
```
Comment (deshraj) [inline embedchain/__init__.py:8]: Can you please make the letters to be lowercase?

Comment (subhajit20) [reply embedchain/__init__.py:8]: sure

Comment (subhajit20) [reply embedchain/__init__.py]: done

**Pull request merged**